packages/create-waku/src/index.ts

```diff
@@ -162,9 +161,7 @@ Options:
 }
 
 async function notifyUpdate() {
-  // keep original require to avoid
-  //  bundling the whole package.json by `@vercel/ncc`
-  const packageJson = createRequire(import.meta.url)('../package.json');
+  const packageJson = await import('../package.json');
```

Comment:
So, is this syntax supported by tsup?

Reply:
yep! I think esbuild is technically what supports it under the hood, but still the answer is yes, it's supported.

Reply:
Actually I was wrong here ^

This errors because of the json import without the attribute, so I just changed it to use `require` which is defined in the banner

the error:

```
TypeError [ERR_IMPORT_ASSERTION_TYPE_MISSING]: Module "file:///.../waku/packages/create-waku/package.json" needs an import attribute of type "json"
```